contrib/agent/build.gradle

```diff
@@ -43,21 +66,37 @@ shadowJar {
 
   // Exclude cruft which still snuck in.
   exclude 'META-INF/maven/**'
+  exclude agentBootstrapClasses
 
   // Relocate third party packages to avoid any conflicts of the agent's classes with the app's
   // classes, which are loaded by the same classloader (the system classloader).
   relocate 'net.bytebuddy', agentRepackaged + '.bytebuddy'
```

Comment:
Byte Buddy itself has a dependency on org.objectweb.asm, which is relocated by Byte Buddy's build process and bundled with the bytebuddy JAR. Also see https://github.com/raphw/byte-buddy/blob/3736a10c170a415c1027d87a5424aca589169710/byte-buddy/pom.xml#L48